ts/features/wallet/component/features/PaymentMethodInitiatives.tsx

```diff
@@ -1,15 +1,20 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
```

Comment:
Are we sure we want this under the wallet feature directory and not under idpay? If yes, why?

Reply:
since we are removing the old "initiative pairing" method in favor of IDPay, it could very well be an occasion to move this and some parent components to `idpay/wallet`